VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -73,8 +73,7 @@ public void updateDrawState(TextPaint ds) {
                     stringRes, new HashMap<String, CharSequence>() {{
                         put("time", formattedTime);
                         put("author", authorSpan);
-                        put("author_label", null == authorLabel ? "" : authorLabel.toUpperCase(
-                                context.getResources().getConfiguration().locale));
+                        put("author_label", null == authorLabel ? "" : authorLabel);
```

Comment:
I think the `null` handling can be remove now, since it's also done at the `ResourceUtil.getFormattedString()` method that's being called here.